rqt_rosmon/src/node_model.cpp

```diff
@@ -173,7 +173,7 @@ QVariant NodeModel::data(const QModelIndex& index, int role) const
 					return int(Qt::AlignRight | Qt::AlignVCenter);
 			}
 			break;
-		case Qt::BackgroundColorRole:
+		case Qt::BackgroundRole:
```

Comment:
I guess we should return `QBrush(QColor(...))` in that case, at least the Qt documentation seems to suggest this.

Reply:
added a commit for that here: https://github.com/lucasw/rosmon/pull/1